src/rastervision/common/models/custom_layers.py

```diff
@@ -0,0 +1,67 @@
+from keras import initializers
```

Comment:
I see you did it for the other two files ... just might want to move it to the top since the stuff we copied spans more than the function that is commented.